lab_1_classify_by_unigrams/main.py

```diff
@@ -11,6 +12,12 @@ def tokenize(text: str) -> list[str] | None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
+    if not isinstance(text, str):
+        return None
+
+    tokens = [symbol.lower() for symbol in text if symbol.isalpha()]
```

Comment:
return as soon as posible, do not create a variable

Reply:
Done